internal/runners/packages/info.go

```diff
@@ -0,0 +1,238 @@
+	logging.Debug("ExecuteInfo")
+
+	language, err := targetedLanguage(params.Language)
+	if err != nil {
+
+		return locale.WrapError(err, fmt.Sprintf("%s_err_cannot_obtain_language", nstype))
+	}
+
+	ns := model.NewNamespacePkgOrBundle(language, nstype)
+
+	pkgName, _ := splitNameAndVersion(params.Package)
+
+	packages, err := model.SearchIngredientsStrict(ns, pkgName)
+	if err != nil {
+		return locale.WrapError(err, "package_err_cannot_obtain_search_results")
+	}
+
+	if len(packages) == 0 {
+		return errs.AddTips(
+			locale.NewInputError("err_package_info_no_packages", `No packages in our catalogue are an exact match for [NOTICE]"{{.V0}}"[/RESET].`, params.Package),
+			locale.Tl("info_try_search", "Valid package names can be searched using [ACTIONABLE]`state search {package_name}`[/RESET]"),
+			locale.Tl("info_request", "Request a package at [ACTIONABLE]https://community.activestate.com/[/RESET]"),
+		)
+	}
+
+	pkg := packages[0]
+
+	authors, err := model.FetchAuthors(pkg.Ingredient.IngredientID, pkg.LatestVersion.IngredientVersionID)
+	if err != nil {
+		return locale.WrapError(err, "package_err_cannot_obtain_authors_info", "Cannot obtain authors info")
+	}
+
+	res := newInfoResult(pkg, authors)
+	out := &infoResultOutput{
+		i.out,
+		res,
+		whatsNextMessages(res.name, res.Versions),
+	}
+
+	i.out.Print(out)
+
+	return nil
+}
+
+// PkgDetailsTable describes package details.
+type PkgDetailsTable struct {
+	Authors   []string `locale:"package_authors,Authors" json:"authors"`
+	Link      string   `locale:"package_link,Link" json:"link"`
+	Copyright string   `locale:"package_copyright,Copyright" json:"copyright"`
+	License   string   `locale:"package_license,License" json:"license"`
+}
+
+type infoResult struct {
+	name            string
+	latestVersion   string
+	Description     string `locale:"," json:"description"`
+	PkgDetailsTable `locale:"," opts:"verticalTable"`
+	Versions        []string `locale:"," json:"versions"`
+}
+
+func newInfoResult(iv *model.IngredientAndVersion, authors model.Authors) *infoResult {
+	res := infoResult{
+		name:          locale.T("unknown_value"),
+		latestVersion: locale.T("unknown_value"),
+		PkgDetailsTable: PkgDetailsTable{
+			Link:      locale.T("unknown_value"),
+			Copyright: locale.T("unknown_value"),
+			License:   locale.T("unknown_value"),
+		},
+	}
+
+	if iv.Ingredient != nil {
+		if iv.Ingredient.Name != nil {
+			res.name = *iv.Ingredient.Name
+		}
+
+		if iv.Ingredient.Description != nil {
+			res.Description = *iv.Ingredient.Description
+		}
+
+		if iv.Ingredient.Links != nil && iv.Ingredient.Links.Self != nil {
+			res.PkgDetailsTable.Link = iv.Ingredient.Links.Self.String()
+		}
+	}
+
+	if iv.LatestVersion != nil {
+		if iv.LatestVersion.Version != nil {
+			res.latestVersion = *iv.LatestVersion.Version
+		}
+
+		if iv.LatestVersion.CopyrightText != nil {
+			res.PkgDetailsTable.Copyright = *iv.LatestVersion.CopyrightText
```

Comment:
Why is this printing as a JSON array? eg.

```
["Copyright (c) Django Software Foundation and individual contributors."]  
```

Reply:
Discussed in Slack: We're not showing this information to users at this time while changes are made in the backend. (Same for the License field.)